### ISSUE TITLE
Create a JupyterLab style widget extension toolbar at the top

### ISSUE THREAD

**Issue by SimonBiggs** [SimonBiggs/scriptedforms#44]
Do away with the floating fab for the kernel reset. Instead use a JupyterLab style toolbar. Much the same as the icon toolbar that exists at the top of a notebook.

It will eventually contain things such as:

* print
* download source
* refresh kernel
* etc.

**Comment by SimonBiggs** [comment]
Heya @robmarkcole, just letting you know that I'm going to prioritise this issue next. I want to update the screenshots on the front page as they are giving misinformation about how to create number and slider variables, but I want the new screenshot to not have the refresh button down the bottom right.

And my work is pretty keen that I get a print button on there. So I'll shift focus to this. If you do have issues with jupyterhub please let me know and once this toolbar is complete I'll shift focus back to jupyterhub.

**Comment by robmarkcole** [comment]
Will do. Btw it would be good to have a file upload button, to enable eg image processing

**Comment by robmarkcole** [comment]
Another idea is a variable viewer, similar to spyder/ matlab one

**Comment by SimonBiggs** [comment]
File upload. ...that is interesting. Can you make that its own issue. I'll have to think about how that would be possible...

**Comment by SimonBiggs** [comment]
With the variable viewer I'll leave that. That will just happen by itself when I turn scriptedforms into a jupyterlab extension.
